manual_tests/test_6_full.yml

```diff
@@ -42,7 +42,7 @@
         instances:
           - host: localhost
             port: 22
-            username: root
+            agent_username: root
```

Comment:
for consistency, maybe `agent_datadog` (or `datadog_agent`) here?

Reply:
same as above, and we want to keep the ``username`` part of it to remain